openapi/platform/platform.yaml

```
@@ -1752,7 +1753,110 @@ paths:
       tags:
         - Options
         - Automation Runs
-  /automations/{id}:
+  /automations:
+    parameters: []
+    get:
+      summary: List Automations
+      responses:
+        '200':
+          description: OK
+          content:
+            application/vnd.api+json:
+              schema:
+                type: object
+                description: A list of automations
+                x-examples:
+                  example-1:
+                    - type: automations
+                      id: Automation-91cb048c-c498-43ba-ab99-2ca2d27d2c46
+                      attributes:
+                        name: Example Automation
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/automations'
+                  links:
+                    type: object
+                    properties:
+                      self:
+                        type: string
+                        format: uri
+                      first:
+                        type: string
+                        description: Provides a link back to the first page of results
+                        format: uri
+                      next:
+                        type: string
+                        description: The URI at which the next batch of purchases can be gotten from
+                        format: uri
+              examples:
+                example-1:
+                  value:
+                    data:
+                      - type: automations
+                        id: Automation-91cb048c-c498-43ba-ab99-2ca2d27d2c46
+                        attributes:
+                          name: Example Automation
+                          trigger: /app/account-group/account-group/created
+                          status: 'on'
+                          tags:
+                            - '["example-tag"]'
```

This is getting double escaped when you preview the docs. 
```suggestion
                            - '["example-tag"]'
```